terracotta/server/rgb.py

```diff
@@ -21,6 +22,36 @@ class RGBQuerySchema(Schema):
     tile_x = fields.Int(required=True, description="x coordinate")
 
 
+def validate_range(data):
+    if isinstance(data, str) and data.startswith("p"):
+        if not re.match("^p\d+$", data):
+            raise ValidationError("Percentile format is `p<digits>`")
+    else:
+        try:
+            float(data)
+        except ValueError:
+            raise ValidationError("Must be a number")
```

Comment:
Can we use the same exception for both branches that contains the expected format?